manifests/sentinel.pp

```diff
@@ -122,20 +150,29 @@
   Stdlib::Absolutepath $log_file = $redis::params::sentinel_log_file,
   String[1] $master_name  = 'mymaster',
   Stdlib::Host $redis_host = '127.0.0.1',
-  Stdlib::Port $redis_port = 6379,
+  Stdlib::Port::Unprivileged $redis_port = 6379,
```

Comment:
I would consider this change (and `sentinel_port` parameter) backwards incompatible, so it's better to stay out of this PR.